<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FeaturesDockWidget</class>
 <widget class="QWidget" name="FeaturesDockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>459</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DirtyDock</string>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QTabBar" name="tabBar" native="true"/>
      </item>
      <item>
       <widget class="QListWidget" name="FeaturesList">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="cbWithin">
        <property name="text">
         <string>Only features fully within the viewport</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header location="global">qtabbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
